# Synchronization
std_msgs/Header header

# Slot ID
int32 id

# Status
bool is_occupied

# Geometric Information
geometry_msgs/Point center
float32 width
float32 length
float32 angle
